packages/react/toast/src/Toast.tsx

```diff
@@ -864,8 +874,16 @@ function getAnnounceTextContent(container: HTMLElement) {
   childNodes.forEach((node) => {
     if (node.nodeType === node.TEXT_NODE && node.textContent) textContent.push(node.textContent);
     if (isHTMLElement(node)) {
-      const isHidden = node.ariaHidden || node.hidden || node.style.display === 'none';
-      if (!isHidden) textContent.push(...getAnnounceTextContent(node));
+      const isHidden =
+        node.ariaHidden ||
+        node.hidden ||
+        node.style.display === 'none' ||
+        node.dataset.radixToastAnnounceExclude === '';
```

Comment:
It bothers me that this is bundled in `isHidden`.
I'd rather we have an extra `const isExcluded` and the do `if (!isHidden & !isExcluded)` later.
Thoughts?

Reply:
Yeah I had something like this originally, sounds good.